docs/products/kafka/kafka-mirrormaker/concepts/disaster-recovery/active-active-setup.rst

```diff
@@ -0,0 +1,34 @@
+Active-active setup
+########################
+
+An active-active setup of MM2 allows for data to be replicated between Cluster A and Cluster B simultaneously. It implies a bidirectional mirroring between the 2 clusters. In order to do this MM2 uses topic prefixes in the form of **<cluster alias>.<topic name>**
+In this setup, there are two actively used Apache Kafka® clusters: K1 and K2. Topic topic exists in both cluster.
+
+.. image:: /images/products/kafka/kafka-mirrormaker/Mirrormaker-Active-Active.png
+    :alt: MirrorMaker 2 Active-Active Setup
+
+Each cluster has it's own producers and consumers. Producers produce to topic. Consumers consume from topic using the same group.id. MirrorMaker 2 replicates data between clusters in both directions, therefore remote topics K1.topic and K2.topic exists in K2 and K1 correspondingly.
```

Comment:
What is K1 and K2? We need to explain
```suggestion
Each cluster has it's own producers and consumers. Producers produce to topic. Consumers consume from topic using the same ``group.id``. MirrorMaker 2 replicates data between clusters in both directions, therefore remote topics ``K1.topic`` and ``K2.topic`` exists in ``K2`` and ``K1`` correspondingly.
```

Reply:
@ftisiot Right above the image there's a sentence that says "In this setup, there are two actively used Apache Kafka® clusters: K1 and K2". K1 and K2 are also indicated in the image as Kafka cluster K1 and Kafka cluster K2.